src/engine/ruleset/integrations/windows/decoders/windows-security.yml

```diff
@@ -167,11 +167,11 @@ normalize:
   # Group: Target User 1 - Set Target User Domain
   - check: array_contains($_ev_target_user_1, $event.id) AND not_exists($user.domain)
     map:
-      - user.domain: $windows.EventData.TargetDomainName
+      - user.domain: parse_fqdn($windows.EventData.TargetDomainName)
```

Comment:
The `parse_fqdn($windows.EventData.TargetDomainName)` statement is used multiple times, would it be better to create a variable to assign it only once and execute the hf `parse_fqdn` only once? what do you think?